Userland/DevTools/HackStudio/HackStudioWidget.cpp

```diff
@@ -461,33 +465,58 @@ void HackStudioWidget::set_edit_mode(EditMode mode)
     m_right_hand_stack->active_widget()->update();
 }
 
-NonnullRefPtr<GUI::Menu> HackStudioWidget::create_project_tree_view_context_menu()
+ErrorOr<NonnullRefPtr<GUI::Menu>> HackStudioWidget::create_project_tree_view_context_menu()
 {
-    m_new_file_actions.append(create_new_file_action("&C++ Source File", "/res/icons/16x16/filetype-cplusplus.png", "cpp"));
-    m_new_file_actions.append(create_new_file_action("C++ &Header File", "/res/icons/16x16/filetype-header.png", "h"));
-    m_new_file_actions.append(create_new_file_action("&GML File", "/res/icons/16x16/filetype-gml.png", "gml"));
-    m_new_file_actions.append(create_new_file_action("P&ython Source File", "/res/icons/16x16/filetype-python.png", "py"));
-    m_new_file_actions.append(create_new_file_action("Ja&va Source File", "/res/icons/16x16/filetype-java.png", "java"));
-    m_new_file_actions.append(create_new_file_action("C Source File", "/res/icons/16x16/filetype-c.png", "c"));
+    auto action = TRY(create_new_file_action("&C++ Source File", "/res/icons/16x16/filetype-cplusplus.png", "cpp"));
+    m_new_file_actions.append(action);
```

Comment:
if we're handling errors properly anyway, lets do this for appending as well:
```cpp
TRY(m_new_file_actions.try_append(action));
```
(and similar for all the other appends below)

Reply:
I was uncertain if I could return from within a function call, as my tests with compiler explorer showed that I couldn't.
But I tries in the code base and it seemed to compile just fine. Cannot say that I know how it works though